platform/core/src/types/PanelModule.ts

```diff
@@ -0,0 +1,30 @@
+import { PubSubService } from "../services";
+
+type Panel = {
+  id: string;
+  name: string;
+  iconName: string;
+  iconLabel: string;
+  label: string;
+  component: React.FC;
+};
+
+type ActivatePanelTriggers = {
+  sourcePubService: PubSubService;
```

Comment:
shouldn't this be `sourcePubSubService`?

Reply:
also having `PanelService.addActivatePanelTriggers` it already has the trigger in the name, maybe we rename the second key to `sourceEvents`?

Reply:
This all makes sense.